releasenotes/notes/AddPodCheck-355c270d2179d511.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The orchestrator check is moving from the Process Agent to the Core Agent. In the next release, this new check will replace the current pod check in the Process Agent. You can start using this new check now by manually setting the environment variable `DD_ORCHESTRATOR_EXPLORER_RUN_ON_NODE_AGENT` to `true`.
```

Changelog uses RST instead of Markdown:
```suggestion
    The orchestrator check is moving from the Process Agent to the Core Agent. In the next release, this new check will replace the current pod check in the Process Agent. You can start using this new check now by manually setting the environment variable ``DD_ORCHESTRATOR_EXPLORER_RUN_ON_NODE_AGENT`` to ``true``.
```